src/domUpdates.js

```diff
@@ -28,19 +30,68 @@ let domUpdates = {
           `
       )
     })
-    // getFavorites();
   },
-  cardButtonConditionals(event) {
-    console.log(event.target)
-    if (event.target.classList.contains('favorite')) {
-      favoriteCard(event);
-    } else if (event.target.classList.contains('card-picture')) {
-      displayDirections(event);
-    } else if (event.target.classList.contains('home')) {
+  viewFavorites(cardArea, user, favButton, currentFavs,) {
+    if (!user.favoriteRecipes.length) {
+      favButton.innerHTML = 'You have no favorites!';
+      return
+    } else {
+      favButton.innerHTML = 'Refresh Favorites'
+      cardArea.innerHTML = '';
+      this.populateCards(user.favoriteRecipes, cardArea, currentFavs)
```

Comment:
Removed the conditionals that did not exist here, the two left do seem to be in a good place here, as the event listener is on the card itself.

Reply:
it was a nice catch to remove the unnecessary conditional, I like the way it looks now.